src/components/UI/UFormSelect/UFormSelect.tsx

```diff
@@ -0,0 +1,38 @@
+interface IFormSelect {
+  title: string;
+  name?: string;
+  required?: boolean;
+  onChange: (option: string) => void;
+}
+
+export type SelectCountryKey = 'country' | 'billing_country';
+
+const items = ['Canada', 'Poland'];
+
+function UFormSelect({ title, name = '', required = true, onChange }: IFormSelect) {
```

Comment:
By the way, you should choose only one variant of defining components, with `const` or with `function`, and use it everywhere. Do not switch between const and function without any significant reason (and the only significant reason to do it is to use `this` :) )